clients/drcachesim/tests/offline-burst_static.templatex

```diff
@@ -20,7 +20,7 @@ DynamoRIO statistics:
 .*
 all done
 Cache simulation results:
-Core #0 \(1 thread\(s\)\)
+Core #0 \(traced CPU\(s\): #0\)
```

Comment:
Even for core-sharded operation, isn't it more useful to know what threads were on a core shard, rather than what original cores were on a core-shard?

Reply:
This request is unrelated to this PR.  This is how the simulators print things out in core-sharded mode.  You can use schedule_stats to see the threads on a core.  For non-replay-as-traced I would agree it isn't that useful because the core number is identical; maybe it should be removed for non-replay-as-traced.  Could file an issue.